app/helpers/question_view_helper.rb

```diff
@@ -13,6 +13,18 @@ def legend(question, page_header, conditional)
     }
   end
 
+  def example_date_in_tax_year_of(date)
+    return Time.zone.today if date.nil?
+
+    year = if date.month > 4 || (date.month == 4 && date.day > 5)
+             date.year
+           else
+             date.year - 1
+           end
```

Comment:
Are you trying to get the current collection year? If so, please us the methods in `CollectionTimeHelper`.

Reply:
ta.